autorally_core/src/arduino/ArduinoOnboard.h

```diff
@@ -44,14 +44,16 @@
 //#include <std_msgs/Float64.h>
 #include <autorally_core/SerialInterfaceThreaded.h>
 #include <autorally_msgs/wheelSpeeds.h>
-#include <autorally_msgs/servoMSG.h>
+#include <autorally_msgs/chassisCommand.h>
 
 #include <boost/tokenizer.hpp>
 //#include <boost/circular_buffer.hpp>
 
 #include <stdio.h>
 #include <string>
 
+#warning autorally_core/ArduinoOnboard.h has been deprecated, refer to autorally_chassis
```

Comment:
Are we going to get this warning every time we compile the code?  That seems unnecessary.